tests/test_gen_pv.py

```diff
@@ -247,8 +249,6 @@ def test_pv_name_error():
 
     # run reV 2.0 generation
     with pytest.raises(KeyError) as record:
-        pp = ProjectPoints(rev2_points, sam_files, 'pv',
```

Comment:
So weird that these lines were left in the test with seemingly no effect. Good catch on cleaning those up